services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/resource_tracker_service_runs.py

```diff
@@ -125,3 +130,44 @@ async def list_service_runs(
         )
 
     return ServiceRunPage(service_runs_api_model, total_service_runs)
+
+
+async def export_service_runs(
+    s3_client: SimcoreS3API,
+    bucket_name: str,
+    user_id: UserID,
+    product_name: ProductName,
+    resource_tracker_repo: ResourceTrackerRepository,
+    wallet_id: WalletID | None = None,
+    access_all_wallet_usage: bool = False,
+    order_by: OrderBy | None = None,  # noqa: ARG001
+    filters: ServiceResourceUsagesFilters | None = None,  # noqa: ARG001
+) -> AnyUrl:
+    started_from = None
+    started_until = None
+    if filters:
+        started_from = filters.started_at.from_
+        started_until = filters.started_at.until
+
+    # Create S3 key name
+    s3_bucket_name = S3BucketName(bucket_name)
+    file_name = f"{datetime.now(tz=timezone.utc).date()}__{uuid4()}.csv"
+    s3_object_key = f"resource-usage-tracker-service-runs/{file_name}"
+
+    # Export CSV to S3
+    await resource_tracker_repo.export_service_runs_table_to_s3(
+        product_name=product_name,
+        s3_bucket_name=s3_bucket_name,
+        s3_key=s3_object_key,
+        user_id=user_id if access_all_wallet_usage is False else None,
+        wallet_id=wallet_id,
+        started_from=started_from,
+        started_until=started_until,
+        order_by=order_by,
+    )
+
+    # Create presigned S3 link
+    generated_url: AnyUrl = await s3_client.create_presigned_download_link(
+        bucket_name=s3_bucket_name, object_key=s3_object_key, expiration_secs=7200
```

Comment:
MINOR: add expiration as part of the app settings or at least add it in a constant in case we want to expose it to the app settings at some point in time